### ISSUE TITLE
Teamspeak doesn't show correct client address.

### ISSUE THREAD

**Issue by Luukth** [chaifeng/ufw-docker#14]
Hello, I'm hosting a teamspeak server using [this panel](https://pterodactyl.io/). When I connect to the teamspeak server my client ip is 172.18.0.1 instead of my public ip. Is there maybe a solution to this problem or could anyone point me in the right direction. I'm by no mean an expert but willing to learn all about new topics.

Output of ifconfig:
```
pterodactyl0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 172.18.0.1  netmask 255.255.0.0  broadcast 172.18.255.255
        inet6 fdba:17c8:6c94::1011  prefixlen 64  scopeid 0x0<global>
        inet6 fe80::42:32ff:fe43:8a1  prefixlen 64  scopeid 0x20<link>
        inet6 fe80::1  prefixlen 64  scopeid 0x20<link>
        ether 02:42:32:43:08:a1  txqueuelen 0  (Ethernet)
        RX packets 1653  bytes 167191 (163.2 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 1658  bytes 133446 (130.3 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```
Added to after.rules
```
# BEGIN UFW AND DOCKER
*filter
:ufw-user-forward - [0:0]
:DOCKER-USER - [0:0]
-A DOCKER-USER -j RETURN -s 10.0.0.0/8
-A DOCKER-USER -j RETURN -s 172.18.0.0/12
-A DOCKER-USER -j RETURN -s 192.168.0.0/16
-A DOCKER-USER -p udp -m udp --sport 53 --dport 1024:65535 -j RETURN

-A DOCKER-USER -j ufw-user-forward

-A DOCKER-USER -j DROP -p tcp -m tcp --tcp-flags FIN,SYN,RST,ACK SYN -d 192.168.0.0/16
-A DOCKER-USER -j DROP -p tcp -m tcp --tcp-flags FIN,SYN,RST,ACK SYN -d 10.0.0.0/8
-A DOCKER-USER -j DROP -p tcp -m tcp --tcp-flags FIN,SYN,RST,ACK SYN -d 172.18.0.0/12
-A DOCKER-USER -j DROP -p udp -m udp --dport 0:32767 -d 192.168.0.0/16
-A DOCKER-USER -j DROP -p udp -m udp --dport 0:32767 -d 10.0.0.0/8
-A DOCKER-USER -j DROP -p udp -m udp --dport 0:32767 -d 172.18.0.0/12
-A DOCKER-USER -j RETURN
COMMIT
# END UFW AND DOCKER
```

![Image of teamspeak](https://i.stack.imgur.com/abeUB.png)

**Comment by chaifeng** [comment]
Hi,

I don't know Teamspeak, but I checked its document. Is your Teamspeak server runing behind a web server? such as Apache2 or Nginx.

If so, you may need changing configurations of your web server, like [mod_remoteip](https://httpd.apache.org/docs/2.4/mod/mod_remoteip.html).[Forward IP Addresses with NGINX Proxy](https://calvin.me/forward-ip-addresses-when-using-nginx-proxy/)

**Comment by Luukth** [comment]
Hi, and thanks for your reply. The TeamSpeak server is not running behind Nginx or Apache2. Yesterday it worked the first time, correct IP was shown, but after recreating the container it shows `172.18.0.1` again.

A couple of minutes ago I tried to configure UFW with your script. It worked the first time, correct IP is shown. Then I recreated/restarted the container and it showed `172.18.0.1`. After that I completely restarted my TeamSpeak client and connected again. Now it shows my correct public IP. 

So for some reason restarting the TeamSpeak client seems to fix the issue. After the client restarts and connecting to the TeamSpeak server the correct IP is shown.

Are there any steps I can perform to find the cause of this problem. Or is this issue more related to TeamSpeak then UFW and docker?
